_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,20 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the https://
 
-When referencing a website, drop the `http://` prefix.
+You don't need to write `http://` or `https://` when referencing a website.
+
+If the site has a URL route without `www` you don't need to write it when you reference it. But keep the `www` if there is only a redirect in place.
+
+You still need to use the full URL when you create a hyperlink.
```

Comment:
We're trying to say, "hey but still use `http://` when you code the link."

Need a way to say this that covers it if they're using an CMS.

So how about:

`Use the full URL for the link destination, for example https://www.dta.gov.au or [other example like https://google.com]`.

Is `link destination` a sufficiently understood concept by users?